deploy/charts/cert-manager/values.yaml

```
@@ -4,6 +4,10 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 global:
+  # Image repository default for all images
+  # +docs:property
+  imageRepository: quay.io/jet-stack
```

suggestion: This doesn't look right to me!

```suggestion
  imageRepository: quay.io/jetstack
```